install.sh

```diff
@@ -166,14 +166,15 @@ DEFAULT_CONNECTORS=(
     tap-fastly
     tap-jira
     tap-kafka
-    #tap-mysql
+    tap-mysql
     tap-pagerduty
     tap-postgres
     tap-s3-csv
     tap-salesforce
     tap-snowflake
     tap-zendesk
-    #target-s3-csv
```

Comment:
Just add tap-sentry but don't uncomment the other. It caused some issues with the build